cltk/stop/middle_high_german/stops.py

```diff
@@ -227,3 +230,18 @@
     "zu",
     "zû",
     "zuo"]
+
+
+class CorpusStoplist(BaseCorpusStoplist):
```

Comment:
@diyclassics Is this your intention, to have a `CorpusStoplist` class in each `stops.py` file? See also Clement's Old Norse version in the same one.

Reply:
Looking at it with fresh eyes—probably not necessary. The advantage of this arrangement would come with any situation in which there is language-specific preprocessing, etc., but this does not appear to be in use at present. (Also, the numpy & scikit learn checks in the language-specific modules seems to be redundant; the super class already does this check. I may remove this.)

Reply:
I like—though I'm not sure this is a universally held preference—that this winds up at the same place:

1. 
- from cltk.stop.stop import BaseCorpusStoplist
- stoplist_builder = BaseCorpusStoplist(language='latin')

&

2.
- from cltk.stop.latin import CorpusStoplist
- stoplist_builder = CorpusStoplist()

Two ways of looking the same thing—the first from a task perspective and the second from a language perspective. (I can see the argument here for renaming BaseCorpusStoplist in the stop.py to the simple CorpusStoplist).